replaydetector/replaydetector.go

```diff
@@ -8,7 +8,14 @@ package replaydetector
 type ReplayDetector interface {
 	// Check returns true if given sequence number is not replayed.
 	// Call accept() to mark the packet is received properly.
-	Check(seq uint64) (accept func(), ok bool)
+	// The return value of accept() indicates whether the accepted packet is
+	// has the latest observed sequence number.
+	Check(seq uint64) (accept func() bool, ok bool)
```

Comment:
@stv0g see commit message / PR body:

> Note: this is not a breaking change as the accept() callback was
previously returning zero values and is now returning one, which can be
ignored if desired. It may cause linter failures in some consumers if
using a strict policy.